CHANGELOG.md

```diff
@@ -3,10 +3,12 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* WDIO bails at the first failure.
```

Comment:
Lets update to reflect this option is open for process.env.TRAVIS || process.env.CI